main/states/busy_state.py

```diff
@@ -69,7 +97,14 @@ def on_enter(self, payload=None):
                 stopAction = StopDetector(self.detection)
                 if classifier[:3] == 'ytd':
                     video_url = reply['identifier']
-                    requests.get('http://localhost:7070/song?vid=' + video_url[4:])
+                    x = requests.get('http://localhost:7070/song?vid=' + video_url[4:])
+                    data = x.json()
```

Comment:
If `requests.get` failed (server 7070 is down, or wrong URL), the response cannot be parsed as JSON and this line will raise an exception.

Consider to catch it here, or it will make SUSI crash.